common/djangoapps/student/views/dashboard.py

```diff
@@ -203,7 +203,7 @@ def _create_recent_enrollment_message(course_enrollments, course_modes):  # pyli
         )
 
 
-def get_course_enrollments(user, org_whitelist, org_blacklist):
+def get_course_enrollments(user, org_whitelist, org_blacklist, load_all_courses=False):
```

Comment:
A lot of times, we add a default argument so that old callers can remain unaffected by new options. But this adds a default argument that changes the behavior of the method. Old callers will suddenly get a truncated list of courses. where they did not expect it in the past. So please either:

A) Make it a non-optional argument so that any callers break early and obviously instead of silently with hard-to-debug data problems later; or...
B) Make the default value for your default argument match the old behavior so that only new callers are aware of the change and can opt into the new behavior.